### PR TITLE
Copy and ember-prism polish 

### PR DESCRIPTION
**Pull request by amyrlam**
## :pushpin: Summary

- Copy polish 
  - Use characters in ember-prism code samples when possible
  - Follow typography guidelines https://design-system-components-hashicorp.vercel.app/components/typography
Add language for a11y 
-This brings the Lighthouse score from 98 to 100 on the index page
- Fix links on index

In working on https://github.com/hashicorp/design-system-components/pull/122, I was going to debug ember-prism here since is open source. But, I ended up getting ember-prism to work in .hbs most of the time, except for `{` and `}` characters, which I can debug separately.

The best way to review this will be to have a look at the webpages as a whole on the Vercel. There were a lot of "on the way" changes as is editing copy, which is why I didn't make more atomic commits. (If you're having trouble with reviewing it and prefer a more atomic commit approach, I'm happy to change that! Just seemed easiest for documentation to do things in one fell swoop. LMK)

Note: Not high priority, review at your leisure

***

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

### CODE REVIEW

File: website/app/templates/index.hbs [hunk 1]
```diff
@@ -86,14 +86,14 @@
...
-      GitHub Repository
+      GitHub repository
     </a>
   </div>
 
 </div>
 
 <section class='ds-section' aria-labelledby='available-icons'>
-  <h2 class='ds-h2 mb-3' id='available-icons'>Available Icons</h2>
+  <h2 class='ds-h2 mb-3' id='available-icons'>Available icons</h2>
```
Comment (amyrlam) [inline website/app/templates/index.hbs:96]: Note: Prettier is not working here, but I've extracted that to a separate issue: https://github.com/hashicorp/flight/issues/365

File: website/app/templates/engineering.hbs [hunk 2]
```diff
@@ -32,33 +32,40 @@
...
 <div class="mt-8 prose prose-lg text-gray-800 pb-6">
-  <h3 class="ds-h3">Examples of Correct Use</h3>
+  <h3 class="ds-h3">Examples of correct use</h3>
   {{! prettier-ignore-start }}
   <CodeBlock
     @language="markup"
-    @code="&lt;button aria-label=&quot;Check Activity&quot;>
- &lt;FlightIcon @name=&quot;activity&quot; />
-&lt;/button>"
+    @code='
```
Comment (didoo) [inline website/app/templates/engineering.hbs:39]: nice!

File: website/app/templates/engineering.hbs [hunk 3]
```diff
@@ -110,18 +119,21 @@
...
-  fill=&quot;currentColor&quot;
-  id=&quot;icon-ember115&quot;
-  width=&quot;16&quot;
-  height=&quot;16&quot;
-  viewBox=&quot;0 0 16 16&quot;
-  xmlns=&quot;http://www.w3.org/2000/svg&quot;>
-  &lt;use href=&quot;/@hashicorp/ember-flight-icons/icons/sprite.svg#alert-circle-16&quot;>&lt;/use>
-&lt;/svg>"
+      @code='
+      <svg
```
Comment (didoo) [inline website/app/templates/engineering.hbs:123]: 😍

**Review by didoo — APPROVED** [review]

**Comment by amyrlam** [comment]
Thanks for the review. Rebased. I approved the Percy, as it was innocuous and seemed like the best use of time

**Pull request merged**